cmd/vmars/draw.go

```diff
@@ -4,73 +4,203 @@ import (
 	"fmt"
 	"image"
 	"image/color"
+	"math"
+	"sort"
 
 	"github.com/bobertlo/gmars"
 	"github.com/hajimehoshi/ebiten/v2"
+	"github.com/hajimehoshi/ebiten/v2/colorm"
 	"github.com/hajimehoshi/ebiten/v2/text/v2"
+	"golang.org/x/image/colornames"
 )
 
-func (g *Game) Draw(screen *ebiten.Image) {
-	warriorColors := make([]ebiten.ColorScale, 3)
-	warriorColors[0].Scale(1, 1, 1, 1)
-	warriorColors[1].Scale(1, 1, 0, 1)
-	warriorColors[2].Scale(0, 1, 1, 1)
+const (
+	SpriteFirst = iota
```

Comment:
Used these markers so that it's easy to add sprites at the beginning and end, without redoing the code.